lib/foreman_puppet.rb

```diff
@@ -1,5 +1,5 @@
 module ForemanPuppet
-  FOREMAN_EXTRACTION_VERSION = '2.6'.freeze
+  FOREMAN_EXTRACTION_VERSION = '2.7'.freeze
```

Comment:
Why? This will be correct as once we extract puppet 2.6 => 3.0 and 3.0 >= 2.7, but it just looks weird.
We can go with 3.0 directly :)

Reply:
bumped into `3.0`, on current develop `SETTINGS[:version].notag` returns `2.6.0`